Projects_To_Do_List.py

```diff
@@ -2,7 +2,13 @@ def Projects_To_Do_List():
     Running = True
     while Running:
         import pickle
-        Projects = pickle.load(open("Projects_To_Do_List/Projects.dat","rb"))
+        try:
```

Comment:
hey everything looks good but can you pls explain me why did you use try-except instead of wb+?

Reply:
No specific reason.